test/module.test.js

```diff
@@ -1960,3 +1965,25 @@ describe('Locale fallback array', () => {
     expect(dom.querySelector('[data-test="de-key"]')?.textContent).toEqual('deKey')
   })
 })
+
+describe('localeProperties object', () => {
+  /** @type {Nuxt} */
+  let nuxt
+
+  beforeAll(async () => {
+    nuxt = (await setup(loadConfig(__dirname, 'basic'))).nuxt
+  })
+
+  afterAll(async () => {
+    await nuxt.close()
+  })
+
+  test('localeProperties object exists and is set to the correct value', async () => {
+    const window = await nuxt.renderAndGetWindow(url('/'))
+    expect(window.$nuxt.$i18n.localeProperties).toEqual({
+      code: 'en',
+      iso: 'en',
+      name: 'English'
+    })
+  })
```

Comment:
Can we attach this test to one of the existing describe blocks? Building a new fixture is time-consuming.

Reply:
i just can't find any appropriate describe block to attach it, what do you think?

Reply:
update: i attached it to the `hreflang` block, all changes are committed.